api/proto/certs/certs.proto

```diff
@@ -0,0 +1,57 @@
+// Copyright(c) 2017-2018 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/certs";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+// This is the request payload for POST /api/v1/edgeDevice/EdgeNodeCerts
+// or /api/v2/edgeDevice/EdgeNodeCerts
+// ZEdgeNodeCertMsg carries a set of X.509 certificate and their properties
+// from EVE to Controller.
+// The message is assumed to be protected by a TLS session bound to the
+// device certificate for v1.
+// The message is assumed to be protected by signing envelope for v2.
+message ZEdgeNodeCertMsg {
+  string deviceUuid = 1;
+  ZCertMsgTypes ztype = 2;
```

Comment:
This message is from the device so why do you need a type which can say it is from the controller??

Reply:
removed ztype totally.